### PR TITLE
Moving CSS Change here to prevent collisions with Fabric Contextualmenu css

### PR DESCRIPTION
**Pull request by joschect**
This change should have been put into the contextual menu rather than the specific place it was put.

### CODE REVIEW

File: src/components/ContextualMenu/ContextualMenu.scss [hunk 1]
```diff
@@ -87,4 +87,27 @@ $ContextualMenu-iconWidth: 24px;
   @include right(8px);
   width: 16px;
   text-align: center;
+}
+
+// The following is temporary to allow the React Contextual Menu to appear correctly.
```
Comment (mikewheaton) [inline src/components/ContextualMenu/ContextualMenu.scss]: Is this comment still relevant? We should avoid including references to internal projects/files/issues where possible.

**Comment by joschect** [comment]
@mikewheaton Good call. I'll remove the comment.

File: src/components/ContextualMenu/ContextualMenu.scss [hunk 2]
```diff
@@ -87,4 +87,23 @@ $ContextualMenu-iconWidth: 24px;
...
+
+.ms-ContextualMenu.ms-FocusZone {
+  position: relative;
+  line-height: 0;
+  border: none;
+  box-shadow: 0px 0px 0px 0px transparent;
+  width: auto;
+}
+
+.ms-Callout.ms-ContextualMenu-Callout{
```
Comment (dzearing) [inline src/components/ContextualMenu/ContextualMenu.scss]: nit space before {

**Comment by dzearing** [comment]
You are adding css, but you mentioned "rather than the specific place it was put". Do you have any classes to remove as well?

**Comment by joschect** [comment]
Yes, but since it's a change in another repo I want to wait till this version is all the way in before I remove it.

**Comment by dzearing** [comment]
Approved

File: src/components/ContextualMenu/ContextualMenu.scss [hunk 3]
```diff
@@ -87,4 +87,18 @@ $ContextualMenu-iconWidth: 24px;
...
+button.ms-ContextualMenu-link:first-child,
+button.ms-ContextualMenu-link:last-child {
+    height: 40px;
+}
+
+.ms-ContextualMenu.ms-FocusZone {
+  position: relative;
+  line-height: 0;
+  border: none;
+  box-shadow: 0px 0px 0px 0px transparent;
```
Comment (dzearing) [inline src/components/ContextualMenu/ContextualMenu.scss]: i'm still not totally sure whats up here; we shouldn't need to apply a box-shadow here; where is it coming from?

**Pull request closed without merging**